tensorbay/cli/ls.py

```diff
@@ -45,8 +46,19 @@ def _echo_data(
         )
 
 
+def _validate_dataset_client(dataset_client: DatasetClientType) -> None:
+    if not dataset_client.status.draft_number and not dataset_client.status.commit_id:
+        click.echo(
+            f'"{dataset_client.name}" has no commit history.',
+            err=True,
```

Comment:
remove the last `,` in this line, and black will format this function into oneline.